packages/token-bridge-sdk/src/hooks/useArbTokenBridge.ts

```diff
@@ -57,10 +62,43 @@ class TokenDisabledError extends Error {
     this.name = 'TokenDisabledError'
   }
 }
+
+function getDefaultTokenName(address: string) {
+  const lowercased = address.toLowerCase()
+  return (
+    lowercased.substring(0, 5) +
+    '...' +
+    lowercased.substring(lowercased.length - 3)
+  )
+}
+
+function getDefaultTokenSymbol(address: string) {
+  const lowercased = address.toLowerCase()
+  return (
+    lowercased.substring(0, 5) +
+    '...' +
+    lowercased.substring(lowercased.length - 3)
+  )
+}
```

Comment:
These two are totally arbitrary, and we can change to something more meaningful if needed.

Reply:
seem fine as is